app/Transformers/Multiplayer/RoomTransformer.php

```diff
@@ -26,7 +26,8 @@ public function transform(Room $room)
         return [
             'id' => $room->id,
             'name' => $room->name,
-            'category' => $room->category,
+            'category' => $room->isRealtime() ? 'realtime' : $room->category,
```

Comment:
Looks like this isn't required. We only ever use this response to check whether it is `Spotlight` or not.